src/scribe_data/checkquery.py

```diff
@@ -159,12 +182,22 @@ def sparql_context(url: str) -> SPARQL.SPARQLWrapper:
     Configure a SPARQL context.
 
     A context allows the execution of SPARQL queries.
-
-    Args:
-        url (str): a valid URL of a SPARQL endpoint.
-
-    Returns:
-        SPARQLWrapper: the context.
+ Parameters
+    -----------
+        source_value : str
+            The source value to find equivalents for (e.g. 'english').
+        source_key : str
+            The source key to reference (e.g. 'language').
+        target_key : str
+            The key to target (e.g. 'iso').
+        error_msg : str
+            The message displayed when a value cannot be found.
+
+    Returns
+    --------
+        The 'target' value given the passed arguments.
```

Comment:
Here we need `SPARQLWrapper : the context.` as that's the return value for this function.